src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    static ArrayList<Task> list = new ArrayList<Task>();
+
+    private static void addTasks(Task a) {
+        list.add(a);
+    }
+    private static void printMarking(int i){
+        System.out.println(String.format(" [%s] [%s] %s",
+                list.get(i).getTypeIcon(), list.get(i).getStatusIcon(), list.get(i).getDescription()));
+    }
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello I am Baymax your personal life assistant \n" + logo);
```

Comment:
Not really a coding standard violation, but if your assistant is called Baymax, why is the ASCII art DUKE?